cli/marathon.py

```diff
@@ -96,8 +96,11 @@ def getGroupDetails(self, data):
                         if 'TCP_PORTS' in app['env']:
                             tcp_ports_value = app['env']['TCP_PORTS']
                             tcp_port_list = json.loads(tcp_ports_value).keys()
+ 
+                        if 'ENABLE_AFFINITY' in app['env']:
```

Comment:
While testing on the haproxy end I found that it needs to be a non empty value. I'll be testing some more. So, stay tuned.